.gitignore

```diff
@@ -22,4 +22,3 @@ etc/data/
 etc/ssl/
 tmp/
 glide.lock
-go.sum
```

Comment:
Why do you remove the this file from .gitignore.
It's generated, do we need to keep track of it?

Reply:
yes for almost every project, they commit go.sum to repo